errors/doc.go

```diff
@@ -15,4 +15,5 @@ limitations under the License.
 */
 
 // Package errors makes a set of error message handlers available for use by Cluster API Providers.
+// Deprecated: This package will be removed in one of the next releases.
```

Comment:
```suggestion
// Package errors makes a set of error message handlers available for use by Cluster API Providers.
//
// Deprecated: This package will be removed in one of the next releases.
```

Deprecated needs to be after a new line to be picked up by tooling AFAIR. Should probably add deprecation to each of the errors in this package and to the migration notes for 1.8

Reply:
golint complains for files other that .doc

Reply:
sorry I wasn't clear with this - I meant the other exported types in the package - e.g. MachineStatusError. I'm not certain that deprecating at the package level is enough for tooling to pick other kinds up as deprecated.